ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -88,28 +104,25 @@ object LedgerApiTestTool {
       sys.exit(0)
     }
 
-    val missingTests = (config.included ++ config.excluded).filterNot(Tests.all(config).contains)
+    val cases: Tests.Tests => Iterable[LedgerTestCase] = m => m.values.flatMap(_.tests)
+    val matches: Iterable[String] => LedgerTestCase => Boolean = prefixes =>
+      test => prefixes.exists(prefix => test.name.startsWith(prefix))
+
+    val allTestCaseNames: Set[String] = cases(Tests.all(config)).map(_.name).toSet
+    val missingTests = (config.included ++ config.excluded).filterNot(prefix =>
+      allTestCaseNames.exists(_.startsWith(prefix)))
     if (missingTests.nonEmpty) {
-      println("The following tests could not be found:")
+      println("The following exclusion or inclusion does not match any test:")
       missingTests.foreach { testName =>
         println(s"  - $testName")
       }
       sys.exit(64)
```

Comment:
Totally unrelated but 64 seems like a weird choice :thinking:

Reply:
I believe @rautenrieth-da picked it, based on [`sysexits`](https://man.openbsd.org/sysexits).